modules/contact/server-ts/resolvers.ts

```diff
@@ -28,10 +29,8 @@ export default () => ({
         });
       } catch (e) {
         log.error(e);
-        return { errors: [{ field: 'serverError', message: t('contact:sendError') }] };
+        throw new Error(t('contact:sendError'));
```

Comment:
Not sure about this. If we decide to jump on Apollo errors handling then we should use the `ApolloError` everywhere (https://www.apollographql.com/docs/apollo-server/features/errors.html#Other-errors). Probably it is better to create some general custom error based on `ApolloError` just like others implemented - https://github.com/apollographql/apollo-server/blob/master/packages/apollo-server-errors/src/index.ts#L169. Or let's each module provides it's own set of errors if it's needed, I don't know.